core/ldap_api.php

```diff
@@ -100,8 +100,33 @@ function ldap_connect_bind( $p_binddn = '', $p_password = '' ) {
 		ldap_log_error( $t_ds );
 	}
 
+	# Set minimum TLS protocol version flag (ex: LDAP_OPT_X_TLS_PROTOCOL_TLS1_2).
+	$t_tls_protocol_min = config_get_global( 'ldap_tls_protocol_min' );
+	if( version_compare( PHP_VERSION, '7.1.0', '>=' ) AND $t_tls_protocol_min > 0 ) {
```

Comment:
By convention, we use `&&` rather than `AND`.

We can avoid an unnecessary config_get_global() call by doing 
```
if( version_compare( PHP_VERSION, '7.1.0', '>=' ) ) {
    $t_tls_protocol_min = config_get_global( 'ldap_tls_protocol_min' );
    if( $t_tls_protocol_min > 0 ) {
```

Reply:
OK done